Training/LoginViewController.m

```diff
@@ -0,0 +1,152 @@
+//
+//  LoginViewController.m
+//  Training
+//
+//  Created by María Eugenia Sakuda on 2/23/15.
+//  Copyright (c) 2015 María Eugenia Sakuda. All rights reserved.
+//
+
+#import "LoginViewController.h"
+#import "UIView+Toast.h"
+#import "InformationAPI.h"
+#import "TTTAttributedLabel.h"
+
+@interface LoginViewController ()
+
+@property (weak, nonatomic) IBOutlet UITextField *emailText;
+@property (weak, nonatomic) IBOutlet UITextField *passwordText;
+@property (weak, nonatomic) IBOutlet UIButton *loginButton;
+//@property (weak, nonatomic) IBOutlet UILabel *termsAndConditionsLabel;
+@property (weak, nonatomic) IBOutlet TTTAttributedLabel *termsAndConditionsLabel;
+@property (weak, nonatomic) IBOutlet UIButton *termsAndConditionsButton;
+@property (weak, nonatomic) IBOutlet UIButton *signUpButton;
+
+
+
+@end
+
+@implementation LoginViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    [self hideElements];
+    [self.view makeToastActivity];
+    
+    NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
+    if([defaults objectForKey:@"email"]){
+        [self performSegueWithIdentifier:@"tabBarSegue" sender:self];
+    }
+    
+    [self.view hideToastActivity];
+    [self showElements];
+    
+    [self.loginButton.layer setBorderWidth:1.0f];
+    [self.loginButton.layer setBorderColor:[UIColor blackColor].CGColor];
+    
+    //Hypervínculos label
+    //[self setLabel];
+    
+    [self setButtonAsLabelLink:self.termsAndConditionsButton];
+    
+    //Is pendent to check if it's already logged
+    // Do any additional setup after loading the view.
+    
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+/*
+#pragma mark - Navigation
+
+// In a storyboard-based application, you will often want to do a little preparation before navigation
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
+    // Get the new view controller using [segue destinationViewController].
+    // Pass the selected object to the new view controller.
+}*/
+
+
+- (IBAction)login:(id)sender {
+    NSString *toastMessage = @"";
+    
+    if(![self isValidEmail: self.emailText.text]){
+        toastMessage = @"Email no válido";
```

Comment:
No ponemos strings en los controller.

La forma de hacerlo es la siguiente: http://www.raywenderlich.com/64401/internationalization-tutorial-for-ios-2014

En definitiva lo que deberia pasar es que tengas los strings en un archivo de internacionalizacion, y desde el controller lo pidas mediante un "key". De esa forma, el sistema se va a encargar de darte el string que corresponda segun el idioma del telefono.

Entiendo que todavia no llegaste a este punto en el tutorial, asi que lo podes postponer.